src/main/java/com/github/tomakehurst/wiremock/client/WireMock.java

```diff
@@ -493,6 +493,10 @@ public static ResponseDefinitionBuilder unauthorized() {
         return aResponse().withStatus(401);
     }
 
+    public static ResponseDefinitionBuilder unauthorizedJson(String body) {
+	    return aResponse().withStatus(401).withHeader(CONTENT_TYPE, "application/json").withBody(body);
```

Comment:
This could be useful for other HttpStatus' too. 

Would it be possible to extract the ```withHeader().withBody()``` out into another method to allow for fluent composition with other error codes? For example unauthorized().withJsonBody() / badRequest().withJsonBody()

Reply:
Yes, sounds better. Updated code with the suggestion.